src/utils.ts

```diff
@@ -39,6 +40,10 @@ export const share = (options: ShareOptions): Promise<void> =>
     });
 
 export type NavigationBarIcon = {
+    /** Identifier. The native side will notify the WebView when the icon is clicked using this id*/
+    id: string;
+    /** URL to be opened by the app as a deep-link if present */
+    url?: string;
```

Comment:
here's where I think we should allow to pass an onPress | url, at least when you invoke useSetNavigationBar where we have the ability to set up the effects via hook.